viral/grosmark_analysis.py

```
@@ -240,37 +290,75 @@ def offline_correlations(
+                trial
+                for trial in session.trials
+                if trial_is_imaged(trial)
+                and (rewarded is None or trial.texture_rewarded == rewarded)
+            ],
+            flu=spks,
+            bin_size=None,
+        )
 
-    shuffled_corrs = get_offline_correlation_matrix(offline, do_shuffle=True, plot=True)
-    real_corrs = get_offline_correlation_matrix(offline, do_shuffle=False, plot=True)
+        shuffled_corrs = get_offline_correlation_matrix(
```

Ruff (0.8.2)

Local variable `shuffled_corrs` is assigned to but never used

Remove assignment to unused variable `shuffled_corrs`

(F841)